pkg/controller/clusters/status_test.go

```diff
@@ -128,11 +293,19 @@ func TestUpdateControlPlaneInitializedCondition(t *testing.T) {
 			err := clusters.UpdateClusterStatusForControlPlane(ctx, client, cluster)
 			g.Expect(err).To(BeNil())
 
-			condition := conditions.Get(cluster, anywherev1.ControlPlaneInitializedCondition)
+			condition := conditions.Get(cluster, tt.wantCondition.Type)
 			g.Expect(condition).ToNot(BeNil())
 
+			wantMessage := tt.wantCondition.Message
+			message := condition.Message
+
+			// Clear these out for easier comparison
 			condition.LastTransitionTime = v1.Time{}
+			condition.Message = ""
+			tt.wantCondition.Message = ""
```

Comment:
why change the test input?

Reply:
I was trying to isolate the message so that we could compare them by substring.

Reply:
is this required if we compare field by field instead of the full condition?

Reply:
no, there is no reason that we can't. this has been updated to directly compare the fields instead.